pyproject.toml

```diff
@@ -10,6 +10,7 @@ packages = [{include = "pipelines"}]
 [tool.poetry.dependencies]
 python = "^3.8.1"
 dlt = {version = "^0.2.7", extras = ["redshift", "bigquery", "postgres", "duckdb"]}
+simple-salesforce = "^1.12.4"
```

Comment:
this should be removed from normal dependencies and placed in the respective group (this is also done!)